pplns-with-job-declaration.tex

```diff
@@ -21,174 +21,193 @@
+	\item share - a hash of a certain difficulty determined by the pool but insufficient to satisfy the networks difficulty, shares are submitted by miners as proof of work to a pool, pools often dynamically set the share difficulty for miners
+	\item job - data given to a miner to hash on, including but not limited to a block template
+	\item work - common term when discussing difficulty of shares, work is often the difficulty of an individual share or a the sum of multiple shares difficulty
+	\item block fees - a portion of every transaction paid to miners
+	\item block subsidy - the predetermined supply issuance of bitcoin in a block, this value halves every 210,000 blocks, the initial subsidy was 50 bitcoin per block
+	\item block reward - block subsidy + block fees
+	\item hashrate - the amount of hashes per second, generally calculated using shares within a time frame, certain machines have "sticker hashrate" which is a general number to indicate the machines level of efficiency for hashing. Commonly represented as Terrahash or Gigahasg per second (TH/s or GH/s)
+\end{enumerate}
 
 \section{Slices}
-In order to pay shares in a fair way with respect to fees, they must be scored somehow and the payout of the block fee must be redistributed accodingly to this score. Doing so, it is necessary to divide them in groups and compare scores only for shares belonging to the same group. Indeed, does not make any sense compare two shares taken randomly in the PPLNS window, because the mempool maximum extractable fees (MMEF) may change consistently. Indeed, it is difficult even to compare shares produced in a single mining round.
+In order to pay shares fairly with Job Declaration, shares must be scored by fees earned in a block template and the payout of the block fees must be distributed accordingly. Doing so, it is necessary to divide them into groups and compare scores only for shares belonging to the same group. Comparing two shares taken randomly in a traditional PPLNS window is inaccurate, because the mempool maximum extractable fees (MMEF) may change consistently. The same is true for randomly selected shares within a mining round.
 
-We call a group of shares whose fees can be scored (and the scores can be compared) a \emph{slice}, which is a portion of the stream of shares. As we seen in the previous paragraph, a slice must not cross two mining rounds. With other words, all shares in one slice must have the same prevhash. It is needed an introduction before giving the precise definition of a slice.
+We call a group of shares whose fees can be scored (and the scores can be compared) a \emph{slice}, which is a portion of the stream of shares. A slice must not cross two mining rounds and all shares in one slice must have the same prevhash. A more precise definition of a \emph{slice}
```

Comment:
A more precise definition of a \emph{slice} ...?
I think you forgot to finish the phrase